test/functional/rpc_zmq.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test for the ZMQ RPC methods."""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+
+
+class RPCZMQTest(BitcoinTestFramework):
+
+    address = "tcp://127.0.0.1:28332"
```

Comment:
nit, inline this (only 2 times)?

Reply:
I actually like it better like this, even if it is only used in two places for now.  And if we add a `addzmqnotification` later, it will be used in more.

But if you strongly prefer this to be inlined, I can of course do it.

Reply:
No strong opinion.